### ISSUE TITLE
Suggestion: provide a droid script for linux/mac without ".sh" on the end.

### ISSUE THREAD

**Issue by nishihatapalmer** [digital-preservation/droid#636]
Although I know it's good form to put `.sh` on the end of shell scripts, it would be really nice to run droid just using `droid` rather than `droid.sh` on linux.

I wonder if we can provide a simple shell script that just calls `droid.sh` with its parameters?  So both versions would work.

I always put something like this in my droid folder.. so might be a good alias for other users?

**Comment by ross-spencer** [comment]
I do something similar with a [todo note](https://gist.github.com/ross-spencer/a58594226e378695680bddead7297659) I maintain locally Matt. I do the following to create a symlink so that I can access it `$ todo` from anywhere:

```bash
$ sudo mv todo.sh /usr/local/bin/  
$ sudo ln -s /usr/local/bin/todo.sh /usr/local/bin/todo
```

Where `todo` becomes `droid.sh` that would be one workaround if you thought it useful.

**Comment by OliverHannan** [comment]
Hi @paulwellsagilej, would be good if you queue this one up next please.

**Comment by paulwellsagilej** [comment]
OK - I'll look at it next

**Comment by tnafrancesca** [comment]
Apparently the sigtool file (in the same folder when you download droid) already has no .sh on the end. There could be some code in there that might make this an easy fix?

**Comment by nishihatapalmer** [comment]
Linux doesnt care about file extensions.  That's a windows thing.  We could just rename the droid.sh script to droid and it would all work.  Or have another script that calls the .sh one to preserve backward compatibility.

It's really just a convention, but a slightly annoying one.

**Comment by steve-daly** [comment]
Yes, this is theoretically as simple as removing the .sh from the end of droid.sh, but we need to maintain compatibility with everyone currently calling droid.sh, so as in the first comment above, having a script called simply 'droid' with the same executable permissions as the .sh files, but which passes on the parameters to droid.sh is worth exploring.
